Cargo.toml

```diff
@@ -38,9 +38,9 @@ mini-moka = "0.10"
 lazy_static = "1.4.0"
 
 # Tycho dependencies
-tycho-core = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-core", tag = "0.42.0" }
-tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-client", tag = "0.42.0" }
-tycho-ethereum = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-ethereum", tag = "0.42.0" }
+tycho-core = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-core", rev = "f8d765554194ddd222e4c6f07811e8c99700615a" }
+tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-client", rev = "f8d765554194ddd222e4c6f07811e8c99700615a" }
+tycho-ethereum = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-ethereum", rev = "f8d765554194ddd222e4c6f07811e8c99700615a" }
```

Comment:
I have to changes these

Reply:
Is there a PR we need to review on tycho-indexer that correlates to this PR then?

Reply:
No it's already merged